var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -34,6 +34,7 @@ class Sqlite(AutotoolsPackage):
             '(unsafe for <3.26.0.0 due to Magellan).')
 
     variant('rtree', default=False, description='Build with Rtree module')
+    variant('colmeta', default=False, description="Build with COLUMN_METADATA")
```

Comment:
I'm leaning towards `column_metadata` for the variant name since that's what the SQLite devs refer to it as